sql_export/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from . import test_sql_query
```

Comment:
I did not know that! I did unit test the same way very recently for a module in v8, and odoo was launching the tests without any troubles.
Is it specific to the v7?

Thanks once again for your help!